freeipa/src/main/java/com/sequenceiq/freeipa/service/polling/clusterproxy/ServiceEndpointHealthListenerTask.java

```diff
@@ -44,7 +44,16 @@ public boolean checkStatus(ServiceEndpointHealthPollerObject serviceEndpointHeal
 
     @Override
     public void handleTimeout(ServiceEndpointHealthPollerObject clusterProxyHealthPollerObject) {
-        throw new CloudbreakServiceException("Operation timed out. Failed to check cluster proxy endpoint health.");
+        String clusterIdentifer = clusterProxyHealthPollerObject.getClusterIdentifier();
+        String errorMessage;
+        try {
+            ClusterProxyRegistrationClient client = clusterProxyHealthPollerObject.getClient();
+            errorMessage = client.readConfig(clusterIdentifer).toString();
+        } catch (Exception e) {
+            errorMessage = String.format("Failed to check cluster proxy endpoint health for %s, error: %s", clusterIdentifer, e.getMessage());
+            LOGGER.debug(errorMessage);
```

Comment:
I think this should be an error level message rather than debug.

Reply:
Thank you! I've changed to LOGGER.error().